bin/create_superuser.sh

```diff
@@ -0,0 +1,11 @@
+# Create superuser
+cat <<EOF | python /app/src/manage.py shell
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+if not User.objects.filter(username="$1").exists():
+    User.objects.create_superuser("$1", "$2", "$3")
+else:
+    print('User "{}" exists already, not created'.format("$1"))
+EOF
```

Comment:
We should probably replace this with a built-in management command in Open Zaak in [this folder](https://github.com/open-zaak/open-zaak/tree/master/src/openzaak/management/commands) and name it something like `create_initial_superuser` (in the same spirit as django's built in `createsuperuser`).

That way, we can do strict CLI arg parsing, now I'm a bit worried with this approach if you have a `"` character in your password for example :-)

The command should check if the user already exists or not, and exit if it does. If it doesn't, it should call the [built in](https://github.com/django/django/blob/master/django/contrib/auth/management/commands/createsuperuser.py) `createsuperuser` command.

Code should be something like:

```py
import os

from django.contrib.auth.management.commands.createsueruser import Command as BaseCommand


class Command(BaseCommand):

    def add_arguments(self, parser):
        super().add_arguments(parser)

        parser.add_argument(
            "--password",
            help="Set the password when the superuser is initially created.",
        )

    def handle(self, **options):
        username = options[self.UserModel.USERNAME_FIELD]
        database = options['database']
        qs = (
            self.UserModel
            ._default_manager.db_manager(database)
            .filter(**{self.UserModel.USERNAME_FIELD: username})
        )
        if qs.exists():
            self.stdout.write("Superuser account already exists, exiting")
            return

        # perform user creation from core Django
        super().handle(**options)

        password = options.get("password", os.environ.get("DJANGO_SUPERUSER_PASSWORD"))
        if password:
            self.stdout.write("Setting user password...")
            user = qs.get()
            user.set_password(password)
            user.save()
```

1. Exits early if the user account already exists -> makes it idempotent
2. adds the ability to set the password (which `createsuperuser` doesn't have yet in this version, while using `--no-input`)
3. Support for envvar `DJANGO_SUPERUSER_PASSWORD`, which newer versions of Django support - we'll already be on the way then for Django 3.2 support without having to update configs. When we update, we can drop the code that sets the password

Building this as a management command gives us the ability to write unit tests as well.